README.md

```diff
@@ -100,6 +99,12 @@ VALUES (:first_name, :last_name, :country)
 ```
 Notice the query annotations (`--! authors`, `--! insert_authors`) and the named bind parameters (`:first_name`, etc.).
 
+1. First of all, you can organize that queries into folder /queries/ as mentioned or pass the folder when running the CLI
+2. You can generate the Rust Code with CLI, for example using a local database 
+```shell
+$ cornucopia live postgres://{usr}:{psw}@localhost:5432/{database}"
+```
```

Comment:
If we're giving an example for a live database, we might as well give one for a managed one too:

"... or an ephemeral database managed by Cornucopia:
```shell
cornucopia schema {path_to_schema}
```
"

or something similar

Reply:
The reason I did not add CLI commands to the README is that I think its better for users to read the `--help` message or the book instead, but I can see why it can be useful to get started.